docs/SDL/GetListOfPermissions/index.md

```diff
@@ -10,75 +10,60 @@ Purpose
-
-4) Display the received “External UCS Status” in the appropriate UI screen (_For more details see the picture ExternalConsentStatus_).   
-
-5) Request the list of permissions per application for which the consent is required (via _GetListOfPermissions(appID)_), when HMI gets the 'user-consent-required' notification per application from SDL (_SDL.OnAppPermissionChanged(appID_, _appPermissionsConsentNeeded: true_)).
+Impotant:  
+When GetListOfPermissions is requested without “appID” parameter, the response of PermissionItem array contains the PermissionGroups (“name” parameters) for all the applications as a whole and “allowed” parameter is common for each group (e.g. if at least one of the applications has disallowed parameter for some group, “allowed” value returned for this group in the context of all application must be false). (_For more details see the diagram GetListOfPermissions without appID_).
+3)	Store the pair of values id<->name of PermissionItem structure which were obtained via GetListOfPermissions response. These pairs will be used for future notifications via [OnAppPermissionConsent](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/docs/SDL/OnAppPermissionConsent/index.md) in case user’s choice for the application to allow some functionality has been updated (_For more details see_ [OnAppPermissionConsent](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/docs/SDL/OnAppPermissionConsent/index.md) (id<->name dependency).
+4)	Display the received “External UCS Status” in the appropriate UI screen (_For more details see the picture ExternalConsentStatus_).
 
 !!! NOTE  
-a) The information about the application (name, ID, etc.) is provided by SDL via either [BasicCommunication.UpdateAppList](../../BasicCommunication/UpdateAppList/index.md) or [BasicCommunication.OnAppRegistered](../../BasicCommunication/OnAppRegistered/index.md) RPCs.  
-b) If HMI has never sent externalConsentStatus before (via [OnAppPermissionConsent](../OnAppPermissionConsent/index.md)), SDL will respond with empty array.  
-c) User can go to settings on HMI and enable or disable "External UCS" for an "entity". HMI will notify SDL accordingly (see figure ExternalConsentStatus).  
+a) The information about the application (name, ID, etc.) is provided by SDL via either BasicCommunication.UpdateAppList or BasicCommunication.OnAppRegistered RPCs.  
+b) If HMI has never sent externalConsentStatus before (via [OnAppPermissionConsent](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/docs/SDL/OnAppPermissionConsent/index.md)), SDL will respond with empty array.  
```

Comment:
should be a relative link: `../onapppermissionconsent`. This currently points to the markdown file on GitHub rather than the page on the dev portal.